lib/packet.ml

```diff
@@ -154,19 +154,23 @@ let write_to_csv ts overdue filename =
   let ecsv = Csv.input_all (Csv.of_string payload) in
   Csv.save filename ecsv
 
+let mac_addr_to_flow s : Flow.t = match s with
+  | "10:10:10:10:10:10" -> A
+  | "20:20:20:20:20:20" -> B
+  | "30:30:30:30:30:30" -> C
+  | "40:40:40:40:40:40" -> D
+  | "50:50:50:50:50:50" -> E
+  | "60:60:60:60:60:60" -> F
+  | "70:70:70:70:70:70" -> G
+  | n -> failwith Printf.(sprintf "Unknown MAC address: %s." n)
+
 let find_flow t =
```

Comment:
@anshumanmohan I think you'll like this change :) 

I found what the parser does to our MAC addresses: 
- start with MAC address `s`
- remove the `:`s from `s`, giving a hexadecimal number
- convert this number to decimal

For example, for flow `A`,
```
10:10:10:10:10:10 -> 101010101010 -> 17661175009296
```
With c8c14564e536eec0dc87d7f26fbe862c7b0c002b, `find_flow` just reverses this procedure and feeds the result to `mac_addr_to_flow`.

Reply:
Gorgeous! Thank you for figuring out this black magic :)

Reply:
Would you now like to explain this black magic in a comment in the code?

Reply:
Done in [c049dd3](https://github.com/cucapra/schedulers-in-ocaml/pull/2/commits/c049dd3f22a39f5390cd2f0f6034fdbbfe74bea1)